ROADMAP.js-ipfs.md

```diff
@@ -0,0 +1,40 @@
+js-ipfs roadmap
+===============
+
+> Roadmap for IPFS implementation in JavaScript. This repo describes the milestones and each of the steps that must be accomplished in order to get js-ipfs ready for people to use
+
+note: libp2p, which is the network stack of IPFS, has its own ROADMAP and should be referenced as `libp2p`, a component, in this project.
+
+## Milestones
+
+#### v0.1.0
+
+> main goal: Working version with a js-ipfs delegated node
+
+- [ ] writable gateway
```

Comment:
i think this (0.1.0) also needs:
- [ ] ipfs node
- [ ] ipfs init / id (keygen)
- [ ] js-dag-store on top of writable gateway

of the core commands https://github.com/ipfs/api/blob/master/level1.md#ipfs-core -- need:
- [ ] ipfs object put / get
- [ ] ipfs refs 
- [ ] ipfs version